### PR TITLE
[4.0] Hide label for domain options in configuration

### PR DESCRIPTION
**Pull request by chmst**
### Summary of Changes
Hide label for email domain options

### Testing Instructions
Look at  the tabdomain options in configuration options.
email-domain-before

The label is superfluous and needs the space. 

This PR hides the label as it is made in the permission tab.

### CODE REVIEW

**Comment by SharkyKZ** [comment]
Please make the changes in `com_users` configuration form instead.

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 09495396829058d0420f2aee5332e2d1e767fd08<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28058">issues.joomla.org/tracker/joomla-cms/28058</a>.</sub>

File: administrator/components/com_users/config.xml [hunk 1]
```diff
@@ -132,6 +132,7 @@
 			name="domains"
 			type="subform"
 			label="COM_USERS_CONFIG_FIELD_DOMAINS_LABEL"
+			hiddenLabel="true"
```
Comment (Quy) [inline administrator/components/com_users/config.xml]: > The four attributes name, type, label and description should be written in this order and at the top of the element definition.

Please move it after description.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 3d1f68e40c3c9bfe3c9819f6243db932833660ad<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28058">issues.joomla.org/tracker/joomla-cms/28058</a>.</sub>

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 3d1f68e40c3c9bfe3c9819f6243db932833660ad<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28058">issues.joomla.org/tracker/joomla-cms/28058</a>.</sub>

**Comment by jwaisner** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28058">issues.joomla.org/tracker/joomla-cms/28058</a>.</sub>

**Pull request merged**